src/akamai/js/akamaiHD_flash.js

```diff
@@ -316,7 +316,25 @@ require("../../../html5-common/js/utils/constants.js");
       if (loaded && !rewind) return;
       else {
         try {
-          this.callToFlash("load("+rewind+")");
+
+          var originalAspectRatio = (9/16);
+
+          // JQuery height detect can fail if a height was not explicitly set.
+          // So we try to find one up the tree, and failing that we use a default common
+          // aspect ratio. Not ideal, but provides viewable result.
+          var wrapperWidth = $(videoItem).parent().parent().width();
+          var wrapperHeight = $(videoItem).parent().height();
+          if (wrapperHeight < 1)
+            wrapperHeight = $(videoItem).parent().parent().height();
+          if (wrapperHeight < 1)
+            wrapperHeight = $(videoItem).parent().parent().parent().height();
+          if (wrapperHeight < 1)
+            wrapperHeight = $(videoItem).parent().parent().parent().parent().height();
```

Comment:
We should add null checks to all these parents. If one does not exist, this will break.